.goreleaser.yaml

```
@@ -29,11 +29,14 @@ changelog:
       - '^test:'
 # for more information on what this target does: https://goreleaser.com/errors/docker-build/
 dockers:
-  - image_templates: ["{{ .Env.REGISTRY }}/open-component-model/{{ .ProjectName }}:{{ .Version }}"]
+  - id: linux-build
+    image_templates:
+    - "{{ .Env.REGISTRY }}/open-component-model/{{ .ProjectName }}:v{{ .Version }}"
```

```suggestion
    - "{{ .Env.REGISTRY }}/open-component-model/{{ .ProjectName }}:{{ .Version }}"
```
`v` is added by default in the version!